assets/src/blocks/Happypoint/HappypointEditor.js

```diff
@@ -38,203 +38,202 @@ export const HappypointEditor = ({ attributes, setAttributes, isSelected }) => {
     override_default_content,
     local_content_provider,
   } = attributes;
-  const [iframeUrl, setIframeUrl] = useState(iframe_url || '');
-  const [embedCode, setEmbedCode] = useState(embed_code || '');
+  const [ iframeUrl, setIframeUrl ] = useState( iframe_url || '' );
+  const [ embedCode, setEmbedCode ] = useState( embed_code || '' );
   const dimensions = { width: 400, height: 100 };
 
-  const { imageUrl } = useSelect(select => {
-    let imageUrl = '';
-    if (id && id > 0) {
-      const imageDetails = select('core').getMedia(id);
-      imageUrl = (imageDetails && imageDetails.source_url) || '';
+  const { imageUrl } = useSelect( ( select ) => {
+    let imageUrlSelect = '';
+    if ( id && id > 0 ) {
+      const imageDetails = select( 'core' ).getMedia( id );
+      imageUrlSelect = ( imageDetails && imageDetails.source_url ) || '';
     }
-    return { imageUrl };
-  }, [id]);
+    return { imageUrlSelect };
```

Comment:
I think that here it should be `return { imageUrl: imageUrlSelect };`

Reply:
Done.